tests/integration/test_smoke.py

```diff
@@ -25,7 +25,7 @@ def test_direct_basic_request():
         req.num_recs,
     )
     # do we get recommendations?
-    assert len(outputs.recs) > 0
+    assert len(outputs.default.articles) > 0
```

Comment:
Not a blocker: Wonder if this might be worth applying some syntactic sugar to avoid having to sprinkle `default` in so many places

Reply:
Quite possibly.